src/api/v1/community_businesses/__tests__/visit_activites.test.integration.ts

```diff
@@ -0,0 +1,152 @@
+import * as Hapi from 'hapi';
+import { init } from '../../../../server';
+import { getConfig } from '../../../../../config';
+import { CommunityBusinesses } from '../../../../models';
+
+
+describe('API v1 :: Community Businesses :: Visit Activities', () => {
+  let server: Hapi.Server;
+  const config = getConfig(process.env.NODE_ENV);
+
+  beforeAll(async () => {
+    server = await init(config);
+  });
+
+  afterAll(async () => {
+    await server.shutdown(true);
+  });
+
+  describe('GET', () => {
+    test(':: successfully gets all activities for a cb', async () => {
+      const res = await (<any> server).inject({
```

Comment:
It complains about the credentials otherwise. I tried to extend the `Hapi.Server` type but couldn't get it working, n thought it was of less importance since it's for a test

Reply:
The reason it was complaining about credentials is because the credentials object you're giving it is the wrong type (see my [comment below](https://github.com/TwinePlatform/twine-api/pull/124#discussion_r214087158))